lgc/builder/SubgroupBuilder.cpp

```diff
@@ -1219,7 +1219,7 @@ Value *SubgroupBuilder::createGroupArithmeticOperation(GroupArithOp groupArithOp
 }
 
 // =====================================================================================================================
-// Create an inline assembly call to cause a side effect (used to workaround mis-compiles with convergent).
+// Create an inline assembly call to cause a side effect (used to workaround miss-compiles with convergent).
```

Comment:
The old one was more correct, but perhaps without the hyphen - miscompile = compile incorrectly.

"work around" should be two separate words as a verb and preposition, as opposed to the noun "workaround".